themes/custom/droopler_theme/droopler_theme.theme

```diff
@@ -400,6 +405,9 @@ function droopler_theme_preprocess_block(&$variables) {
 function droopler_theme_is_special_route() {
   $route = \Drupal::routeMatch();
   $route_name = $route->getRouteName();
+  if ($route_name == 'entity.commerce_product.canonical') {
```

Comment:
Też dałbym to do modułu, a w skórce ewentualnie załatwił to np. przez drupal_alter

Reply:
(tu ta sama uwaga co do tytułów, kiedyś będziemy musieli to zoptymalizować, bo modułów dodatkowych będzie przybywać)